src/components/ArticleList.jsx

```diff
@@ -1,18 +1,17 @@
 import { useState, useEffect } from 'react'
-import axios from "axios"
-import { ErrorComponent } from './ErrorComponent';
-import { FancyBorder } from './FancyBorder';
+import { ErrorComponent } from './ErrorComponent'
+import { ArticleCard } from './ArticleCard'
+import { fetchArticles } from './api'
 
 export function ArtcileList() {
     const [articles, setArticles] = useState([])
     const [isLoading, setIsLoading] = useState(true)
-    const [error, setError] = useState(null);
+    const [error, setError] = useState(null)
 
     useEffect(() => {
-        let endpoint = 'https://nc-news-7miy.onrender.com/api/articles'
-        axios.get(endpoint)
```

Comment:
Don't forget to set isLoading to true at the beginning of the useEffect, in case error occurs and require re-rendering